src/main/resources/reference.conf

```diff
@@ -66,35 +66,17 @@ sparkz {
     messageLengthBytesLimit = 16777216
 
     # String with IP address and port to send as external address during handshake. 
-    # Could be set automatically if UPnP is enabled.
     #
     # If `declared-address` is set, which is the common scenario for nodes running in the cloud, 
     # the node will just listen to incoming connections on `bindAddress:port` and 
     # broadcast its `declaredAddress` to its peers. 
-    # UPnP is supposed to be disabled in this scenario.
     #
-    # If declared address is not set and UPnP is not enabled, the node will not listen to incoming connections at all.
-    #
-    # If declared address is not set and UPnP is enabled, the node will attempt to connect to an IGD, retrieve its
-    # external IP address and configure the gateway to allow traffic through. If the node succeeds, the IGD's external
-    # IP address becomes the node's declared address.
-    #
-    # In some cases, you may both set `decalredAddress` and enable UPnP (e.g. when IGD can't reliably determine its
-    # external IP address). In such cases the node will attempt to configure an IGD to pass traffic from external port
-    # to `bind-address:port`. Please note, however, that this setup is not recommended.
+    # If declared address is not set, the node will not listen to incoming connections at all.
     # declaredAddress = ""
 
-    # Enable UPnP tunnel creation only if you router/gateway supports it. Useful if your node is runnin in home
-    # network. Completely useless if you node is in cloud.
-    upnpEnabled = no
```

Comment:
take a look at `src/main/resources/scripts/twinschainConfigGenerator.sh` file.
Need to clean the same configs there.

Reply:
good catch, I've missed it